slack-api-model/src/main/java/com/slack/api/model/block/element/RichTextSectionElement.java

```diff
@@ -146,7 +150,23 @@ public static class TextStyle {
         private boolean bold;
         private boolean italic;
         private boolean strike;
+        private boolean highlight;
```

Comment:
these are not mentioned in the documents but actually these properties work for all elements

Reply:
🤔 this makes sense, from what I was able to find there are currently only 2 versions of these style objects